### ISSUE TITLE
pointerdown fire only once on Firefox linux x64

### ISSUE THREAD

**Issue by padewitte** [jquery/PEP#252]
Pointer down is only fire once on firefox with jquery/pep 0.4.0.
Same test is working well with jquery/pep 0.3.0

I only test firefox on linux x64. Chrome and Safari are working well.

I use the here to reproduce the problem https://gist.github.com/padewitte/691df62a2d2b4fccd445
Just change the version bac to 0.3.0, click pointerdown and border color change each time you click our touch the screen in the svg elements.

**Comment by scottgonzalez** [comment]
Duplicate of #242.
